webapp/channels/src/packages/mattermost-redux/src/utils/post_list.ts

```diff
@@ -79,6 +81,20 @@ export function makeFilterPostsAndAddSeparators() {
                     continue;
                 }
 
+                // Filter out burn-on-read posts when disabled or expired
+                if (post.type === Posts.POST_TYPES.BURN_ON_READ) {
+                    // Skip if feature is disabled
+                    if (!isBorEnabled) {
```

Comment:
We don't have to stop displaying existing BoR posts if feature is disabled, and only restrict their creation. There was a discussion on this in this week's sync call about the same.

Reply:
thanks for the feedback. This is fixed now 👍